zerver/views/documentation.py

```diff
@@ -146,13 +146,27 @@ def add_integrations_context(context: Dict[str, Any]) -> None:
     context['integrations_dict'] = alphabetical_sorted_integration
     context['integrations_count_display'] = integrations_count_display
 
+def add_integrations_open_graph_context(context: Dict[str, Any], request: HttpRequest) -> None:
+    path_name = request.path.rstrip('/').split('/')[-1]
+    if path_name in INTEGRATIONS:
+        integration = INTEGRATIONS[path_name]
+        context['OPEN_GRAPH_TITLE'] = 'Zulip Integration - %s' % (integration.display_name,)
+        context['OPEN_GRAPH_DESCRIPTION'] = (
+            'Instructions to configure a Zulip integration for %s' % (integration.display_name,))
+
+    elif path_name in CATEGORIES:
+        category = CATEGORIES[path_name]
+        context['OPEN_GRAPH_TITLE'] = 'Zulip Integrations - %s' % (category,)
```

Comment:
How about
`Connect your {category} tools to Zulip`
Doesn't work for all our categories, but we can fix forward if we start seeing a lot of links to these on twitter. 

The description can be the same as the one I suggested for the individual integration pages.

Reply:
Fair enough!